### PR TITLE
Feature: Flow 6.0 compatibility

### PR DESCRIPTION
**Pull request by daniellienert**

### CODE REVIEW

**Comment by sbruggmann** [comment]
@daniellienert Thanks for your PR.

For PSR-7 in Flow 6.0 is another change needed: https://github.com/daniellienert/Wwwision.GraphQL/compare/task/support-flow-6...sbruggmann:task/support-flow-6

**Comment by bwaidelich** [comment]
@daniellienert Thanks a lot for this. You marked the PR "WIP". can you recall what's still missing?

File: composer.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
     "name": "wwwision/graphql",
     "license": "MIT",
     "require": {
-        "neos/flow": "^4.0 || ^5.0",
+        "neos/flow": "^4.0 || ^5.0 || ^6.0",
```
Comment (bwaidelich) [inline composer.json]: Is that version really compatible with 6.0 *and* below??

Comment (bwaidelich) [reply composer.json]: I changed the dependencies to support 6.0+ only

**Comment by bwaidelich** [comment]
I solved the conflicts, included Stefans change, limited composer dependencies and tested this briefly.

Seems to work so I'll go ahead and tag a new version (3.2.0)

@daniellienert & @sbruggmann thank you very much! :)

**Pull request merged**